common/lib/capa/capa/templates/codeinput.html

```diff
@@ -4,9 +4,7 @@
 from openedx.core.djangolib.markup import HTML
 %>
 <div id="textbox_${id}" class="capa_inputtype textbox cminput">
-  % if response_data['label']:
-      <label class="problem-group-label" for="cm-textarea-${id}">${response_data['label']}</label>
-  % endif
+  <label class="sr problem-group-label" for="cm-textarea-${id}">Code Editor ${response_data['label']}</label>
```

Comment:
@staubina `response_data['label']` is coming from [capa_problem.py](https://github.com/edx/edx-platform/blob/master/common/lib/capa/capa/capa_problem.py#L812). As far as i remember empty string value of `response_data['label']` causes accessibility and styling issues.